e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -75,19 +90,48 @@ const utilisationReportPage = {
   utilisationTab: {
     downloadReportLink: () => cy.get('[data-cy="download-report-link"]'),
     table: {
-      row: (feeRecordId) => cy.get(`tr[data-cy="utilisation-table-row-${feeRecordId}"]`),
+      // NOTE: Using the ^= selector to match all table rows starting with the given data-cy value (regardless of the actual fee record id value).
+      rows: () => cy.get(`tr[data-cy^="utilisation-table-row-"]`),
+      /**
+       * NOTE: this selector should only be used with test data with unique exporters.
+       * If fee record id is known then the preference is to use that selector as it has guaranteed uniqueness.
+       */
+      rowWithExporter: (exporter) => ({
+        facilityId: () => cy.get(`tr[data-cy*="utilisation-table-row--exporter-${exporter}"]`).find('[data-cy="facility-id"]').first(),
```

Comment:
Is `.first()` required? it looks like it might be needed 

Assuming the `exporter` is an ID, can we rename to `exporterId`?

Reply:
Should it be `exporterName`? Just the current param name looks like it would be an object

Do we need the `.first()`'s here?

Reply:
It shouldn't be `exporterName` because we use `exporter` extensively in utilisation reports and it always is just a string not an object in this context so it would be inconsistent to do it differently here and a very large effort task to rename it generally to `exporterName`.

I thought the `first`s were necessary as thought it returned an array but seems it id not required if there is only one to find so have removed 👍

Reply:
Thanks @RLavender98 

It's not clear what `exporter` actually is, would like to rename throughout - if it's not an exporter name or ID, curious what it actually is 🤔 

But not a blocker for this PR, better to be consistent